### ISSUE TITLE
Error "$event.path is undefined" in FireFox

### ISSUE THREAD

**Issue by PeterRenken** [glutengo/angular-mgl-timeline#29]
Using the timeline, we experience an error in firefox when clicking on a timeline entry. The error text in the console is:
```
ERROR TypeError: "$event.path is undefined"
    toggle Angular
View_MglTimelineEntryComponent_0 MglTimelineEntryComponent.html:3 handleEvent13Angular
```
The FireFox version is 65.0.1 running on Ubuntu 16.04 and 18.04
The angular-mgl-timeline version is 0.3.0.

**Comment by glutengo** [comment]
should be fixed in v 0.3.2 thanks to this PR: #26

Please give me some feedback if this works for you

**Comment by PeterRenken** [comment]
Component version 0.3.2 fixes the problem.
Thanks very much.
